docs/sources/mimir/set-up/migrate/migrate-from-thanos-or-prometheus.md

```diff
@@ -69,6 +69,63 @@ Blocks from Prometheus don't have any external labels stored in them.
 Only blocks from Thanos use labels.
 {{< /admonition >}}
 
+{{< admonition type="note" >}}
+If you encounter the HTTP error 413 "Request Entity Too Large" when uploading blocks using mimirtool, and if Nginx is being used as a reverse proxy, the uploaded block size may be exceeding Nginx's default maximum allowed request body size.
```

Comment:
We could make this note a bit more active.

" . . . and if you're using Nginx as a reverse proxy, the uploaded block size exceeds Nginx's defailt maximum allowed request body size."